k8s-deployment/todo-app-service.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   namespace: todo-app
   labels:
     name: "todo-app"
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "3000"
```

Comment:
Shouldn't these annotations in the deployment file? You probably want to monitor all instances of the deployment and not only a single instance at time?